pages/profile/permission/index.js

```diff
@@ -22,8 +22,16 @@ module.exports = settings => {
   app.use('/', (req, res, next) => {
     const hasRoles = some(req.profile.roles, role => role.establishmentId === req.establishmentId);
     const hasPil = req.profile.pil && req.profile.pil.status === 'active' && req.profile.pil.establishmentId === req.establishmentId;
-    const hasProjects = some(req.profile.projects, project => project.status === 'active' && project.establishmentId === req.establishmentId);
-    res.locals.static.isNamed = !!hasRoles || !!hasPil || !!hasProjects;
+
+    const hasProjects = some(req.profile.projects, project => {
+      return project.status === 'active' && project.establishmentId === req.establishmentId;
+    });
+
+    const hasAdditionalProjects = some(req.profile.projects, project => {
+      return project.status === 'active' && project.additionalEstablishments.map(est => est.id).includes(req.establishmentId);
+    });
+
+    res.locals.static.isNamed = !!hasRoles || !!hasPil || !!hasProjects || !!hasAdditionalProjects;
```

Comment:
I'd prefer the cast to bool on assignment if we're using the "is/has" naming convention, but as it was already like this I'll ignore it.